src/solver/edges/IngredientEdge.tsx

```diff
@@ -106,13 +110,32 @@ export const IngredientEdge: FC<EdgeProps<Edge<IIngredientEdgeData>>> = ({
           }}
           className="nodrag nopan"
         >
-          <Group gap="4px">
-            <FactoryItemImage size={16} id={data?.resource.id} />
-            <Text size="10px">
-              <RepeatingNumber value={data?.value} />
-              /min
-            </Text>
-          </Group>
+          <Tooltip
+            color="dark.8"
+            label={
+              maxBelt && (
```

Comment:
So, here we will have a `selectedBelt` instead of using always `maxBelt`; and furthermore we know that the belt will be always available so no need for undefined check 😄